scripts/commands.sh

```diff
@@ -76,64 +75,72 @@ doctor() {
 }
 
 make-shell() {
-    sha=$(git -C $DIR rev-parse origin/master)
-    log "Using commit $sha for this development shell"
-    url="https://github.com/Caascad/toolbox/archive/${sha}.tar.gz"
-    log "Calculating sha256 for $url"
-    sha256=$(nix-prefetch-url --unpack $url 2>/dev/null)
-    # Check if requested tools are available
-    instantiate_args=""
-    for tool in "$@"
-    do
-        instantiate_args="$instantiate_args -A $tool"
-    done
-    result=$(nix-instantiate $url $instantiate_args 2>&1 || true)
-    if echo $result | grep -q error; then
-        log-error "Error: '$(echo $result | sed "s/.*attribute '\([^']*\)'.*/\1/")' is not available in the toolbox"
-        exit 1
-    fi
+    log "Creating shell ..."
+    commit=$(_lastCommit)
+    _generateToolboxJSON $commit
```

Comment:
commit is not used later
```suggestion
    _generateToolboxJSON $(_lastCommit)
```

Reply:
Done